android_tests/lib/android/specs/common/command.rb

```diff
@@ -9,45 +9,45 @@ def before_first
   end
 
   t 'check all command no arg' do
-    Selenium::WebDriver::Remote::Bridge.method_defined?(:shake).must_equal true
-    Selenium::WebDriver::Remote::Bridge.method_defined?(:launch_app).must_equal true
-    Selenium::WebDriver::Remote::Bridge.method_defined?(:close_app).must_equal true
-    Selenium::WebDriver::Remote::Bridge.method_defined?(:reset).must_equal true
-    Selenium::WebDriver::Remote::Bridge.method_defined?(:device_locked?).must_equal true
-    Selenium::WebDriver::Remote::Bridge.method_defined?(:open_notifications).must_equal true
-    Selenium::WebDriver::Remote::Bridge.method_defined?(:toggle_airplane_mode).must_equal true
-    Selenium::WebDriver::Remote::Bridge.method_defined?(:current_activity).must_equal true
-    Selenium::WebDriver::Remote::Bridge.method_defined?(:current_package).must_equal true
-    Selenium::WebDriver::Remote::Bridge.method_defined?(:get_network_connection).must_equal true
-    Selenium::WebDriver::Remote::Bridge.method_defined?(:get_performance_data_types).must_equal true
-    Selenium::WebDriver::Remote::Bridge.method_defined?(:get_performance_data).must_equal true
-    Selenium::WebDriver::Remote::Bridge.method_defined?(:get_system_bars).must_equal true
-    Selenium::WebDriver::Remote::Bridge.method_defined?(:get_display_density).must_equal true
-    Selenium::WebDriver::Remote::Bridge.method_defined?(:is_keyboard_shown).must_equal true
+    Selenium::WebDriver::Remote::OSS::Bridge.method_defined?(:shake).must_equal true
+    Selenium::WebDriver::Remote::OSS::Bridge.method_defined?(:launch_app).must_equal true
+    Selenium::WebDriver::Remote::OSS::Bridge.method_defined?(:close_app).must_equal true
+    Selenium::WebDriver::Remote::OSS::Bridge.method_defined?(:reset).must_equal true
+    Selenium::WebDriver::Remote::OSS::Bridge.method_defined?(:device_locked?).must_equal true
+    Selenium::WebDriver::Remote::OSS::Bridge.method_defined?(:open_notifications).must_equal true
+    Selenium::WebDriver::Remote::OSS::Bridge.method_defined?(:toggle_airplane_mode).must_equal true
+    Selenium::WebDriver::Remote::OSS::Bridge.method_defined?(:current_activity).must_equal true
+    Selenium::WebDriver::Remote::OSS::Bridge.method_defined?(:current_package).must_equal true
+    Selenium::WebDriver::Remote::OSS::Bridge.method_defined?(:get_network_connection).must_equal true
+    Selenium::WebDriver::Remote::OSS::Bridge.method_defined?(:get_performance_data_types).must_equal true
+    Selenium::WebDriver::Remote::OSS::Bridge.method_defined?(:get_performance_data).must_equal true
+    Selenium::WebDriver::Remote::OSS::Bridge.method_defined?(:get_system_bars).must_equal true
+    Selenium::WebDriver::Remote::OSS::Bridge.method_defined?(:get_display_density).must_equal true
+    Selenium::WebDriver::Remote::OSS::Bridge.method_defined?(:is_keyboard_shown).must_equal true
```

Comment:
Probably unnecessary but what are your thoughts on cleaning this up a bit? Something like 
```ruby
    {
        shake: true,
        launch_app: true,
        close_app: true,
        reset: true,
        device_locked?: true,
        open_notifications: true,
        toggle_airplane_mode: true,
        current_activity: true,
        current_package: true,
        get_network_connection: true,
        get_performance_data_types: true,
        get_performance_data: true,
        get_system_bars: true,
        get_display_density: true,
        is_keyboard_shown: true
    }.each { |method_name, v|
      Selenium::WebDriver::Remote::OSS::Bridge.method_defined?(method_name).must_equal v
    }
```